enterprise/frontend/src/metabase-enterprise/sandboxes/components/GTAPModal.jsx

```diff
@@ -109,7 +107,7 @@ export default class GTAPModal extends React.Component {
         ? error.data
           ? error.data.message || JSON.stringify(error.data)
           : JSON.stringify(error)
-        : UNKNOWN_ERROR_MESSAGE;
+        : t`Unknown error encountered`;
```

Comment:
Do we already have this string in the translation files or is this going to be added in the new batch of strings for 44?